test/defaults.test.ts

```diff
@@ -121,4 +121,30 @@ describe("graphql.defaults()", () => {
       expect(result).toStrictEqual(mockData);
     });
   });
+
+  it.only("handle baseUrl set with /api/v3 suffix", () => {
```

Comment:
```suggestion
  it("handle baseUrl set with /api/v3 suffix", () => {
```

Reply:
Ohhh good catch. Interesting that coverage was 100% even when only this test was run 🤔 Maybe I should add a linting rule from preventing `it.only` statements to be merged